pyam/core.py

```diff
@@ -1284,6 +1287,33 @@ def export_meta(self, excel_writer, sheet_name='meta'):
         if close:
             excel_writer.close()
 
+    def to_datapackage(self, path='datapackage.zip', tmp='tmp'):
+        """Export object to a
```

Comment:
... "Data Package" missing?

Reply:
sorry about this, the docstring got lost during committing